directory_sso_api_client/version.py

```diff
@@ -1 +1 @@
-__version__ = '1.0.1'
+__version__ = '1.0.2'
```

Comment:
semver would be 1.1.0, because we've added a new feature. (1.0.2 would indicate bug fix).